modules/centralized-logging/variables.tf

```diff
@@ -40,6 +40,11 @@ variable "bucket_name" {
   type        = string
 }
 
+variable "parent_folder" {
+  description = "The folder id that will be logged."
```

Comment:
description is kinda confusing.  I think it's meant to be something like.

The folder id that will be configured so all its projects will add data logs for every resource.  (or something to that effect...I think?)

Reply:
Changed the description, I think that your suggestion make more sense.